src/controller/BallotController.ts

```diff
@@ -64,4 +64,16 @@ export default {
       res.status(204).send()
     }
   ),
+  getVoteResult: asyncHandler(
+    async (req: Request<{ ballotID: string }, {}, {}>, res) => {
+      const result = await ballotHandler.getVoteResult(req.params.ballotID);
+      res.status(200).send(result);
```

Comment:
Idk why but I think we agreed on always returning an object with `body: ...`
```suggestion
      res.status(200).send({
        body: result
      });
```

Reply:
Resolved in [#52 Using object with res.send in BallotController](https://github.com/LeoTuet/backend-we-want-to-sleep/pull/54/commits/91c0f570c9b931ffdb31633babb6f062b19a1f1e)